metaconfig-core/shared/src/main/scala/metaconfig/Conf.scala

```diff
@@ -33,7 +34,7 @@ sealed abstract class Conf extends Product with Serializable {
     ConfGet.get(this, path, extraNames: _*)
   def getOrElse[T](path: String, extraNames: String*)(
       default: T
-  )(implicit ev: ConfDecoder[T]): Configured[T] =
+  )(implicit ev: ConfDecoderWithDefaultMaybe[T]): Configured[T] =
     ConfGet.getOrElse(this, default, path, extraNames: _*)
```

Comment:
do you need to change the implicit for this call as well? will `ConfGet.getOrElse` use the `default` parameter in its first matching case (the one currently not using default)?

Reply:
Hm, I don't found usage of this method in macros. But I would leave the defaults support here for uniformity with `getSettingOrElse` method which has same functionality

Reply:
i meant `ConfGet.getOrElse`, it's used by `getSettingsOrElse` (which is used in macros).

Reply:
Then I don't understand the question. `ConfGet.getOrElse` [uses](https://github.com/scalameta/metaconfig/pull/92/files#diff-9aff91f762e9b2b1da37ce46226c0373R24-R25) default parameter in its first matching case

Reply:
the question is: you changed `ev` for `Conf.getOrElse` and `Conf.getSettingOrElse` to be `ConfDecoderWithDefaultMaybe`; but _i think_ you didn't make the same change in the only method they call (`ConfGet.getOrElse`). so either you don't need `WithDefaultMaybe` in this file, or you *do* need it in `ConfGet`.